exonum-java-binding/core/rust/src/runtime/service_factory_adapter.rs

```diff
@@ -58,7 +58,12 @@ impl JavaServiceFactoryAdapter {
                 system_lib_path: Some(absolute_library_path()),
             };
 
-            let runtime = JavaServiceRuntime::new(config, internal_config);
+            let vm = JavaServiceRuntime::create_java_vm(
```

Comment:
Why this change in the production code? Isn't this method supposed to be test-only?

Reply:
Rewritten with an additional method for integration tests.